### ISSUE TITLE
Mistake in documentation

### ISSUE THREAD

**Issue by RakeshPotnuru** [github/training-kit#851]
## Overview
**TL;DR**
Is this a mistake in the documentation [here](https://github.com/git-guides/git-add)?

### Extra Detail
Please see the red indications.

#### Screenshots
<img width="608" alt="Screenshot 2021-08-12 092956" src="https://user-images.githubusercontent.com/70439799/129146410-0c67a315-346f-43d4-b98f-a17e0ceea26f.png">

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://lab.github.com/githubtraining/introduction-to-github) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.

### Questions?
- Leave a comment on this issue! Make sure to use `@mentions` if you want a specific person's attention!

**Comment by aniruddha-5** [comment]
I would like to take up this issue. I am participating for Hacktoberfest. Will be more than happy to help.

**Comment by ppeters0502** [comment]
I'd like to work on this issue for Hacktoberfest, I'll fork the repository and make the change!

**Comment by ppeters0502** [comment]
Nevermind, looks like the markdown file has already been [updated](https://github.com/github/training-kit/commit/f9a9a2df9ab838c7edfcf3eaf1b95419cbe022ba), a new build just needs to be pushed out and then the changes should show up.
